backend/cmmgen.ml

```diff
@@ -415,12 +415,16 @@ let exttype_of_sort (s : Jkind.Sort.const) =
   match s with
   | Value -> XInt
   | Float64 -> XFloat
+  | Word -> XInt
+  | Bits32 -> XInt32
+  | Bits64 -> XInt64
   | Void -> Misc.fatal_error "Cmmgen.exttype_of_sort: void encountered"
 
 let machtype_of_sort (s : Jkind.Sort.const) =
   match s with
   | Value -> typ_val
   | Float64 -> typ_float
+  | Word | Bits32 | Bits64 -> typ_int
```

Comment:
I can't leave a comment in the right place, but I'm wondering if, since we've decided it's fine to use `typ_int` for all these cases, we should delete `Cmm_helpers.typ_int64`.  It is only used in one place (`To_cmm_shared.machtype_of_kind`) and I think that use can now be replaced with `typ_int`.  @mshinwell?

Reply:
Latest commit in this PR removes both `Cmm_helpers.typ_int64` and `Cmm_helpers.Extended_machtype.typ_int64` as discussed in our meeting, and there seems to already be an assert in `cmm_helpers.ml` checking `size_int = 8`:

https://github.com/ocaml-flambda/flambda-backend/blob/dad7b3a289307c83943aec3a91693e6f781599ff/backend/cmm_helpers.ml#L1481